### PR TITLE
CSV for 2014 county-level results

### PR DESCRIPTION
**Pull request by ssdatar**
Hi Derek,

Here's the cleaned up CSV file for county-level results for the 2014 election, per to your requirements. Please let me know if this works for you or if I need to make any changes.

--
Saurabh

### CODE REVIEW

**Comment by dwillis** [comment]
Hey @ssdatar, thanks! Two things: can you downcase the headers and see if there's a way to include the district number for U.S. House races?

**Comment by ssdatar** [comment]
I'll change the headers, but there were no district numbers in the original pdfs. I included the empty column because all other states had them empty. I can write a small script for it. Are [these](http://www.njleg.state.nj.us/districts/districtnumbers.asp) the correct numbers? Just want to make sure I don't put in anything wrong.

**Comment by dwillis** [comment]
Thanks - the districts are in the originals, but in the office title: "First Congressional District", which would be 1, for example. There are 13, I think, so if it's hard to parse that, don't worry about it.

**Comment by ssdatar** [comment]
Ok I think I will give it a shot

**Comment by ssdatar** [comment]
Done!

**Comment by dwillis** [comment]
Thanks!
